crates/gossip/src/respond/harness.rs

```diff
@@ -38,19 +39,20 @@ impl RespondTestHarness {
             .await
             .unwrap();
 
-        #[derive(Debug)]
-        struct NoopHandler;
-
-        impl TxBaseHandler for NoopHandler {}
-        impl TxHandler for NoopHandler {}
-
-        let transport = builder
-            .transport
-            .create(builder.clone(), Arc::new(NoopHandler))
-            .await
-            .unwrap();
-
-        let (tx, rx) = tokio::sync::mpsc::channel(1);
+        let (tx, rx) = tokio::sync::mpsc::channel(100);
```

Comment:
Now that the way the gossip module handles sending messages has changed, we're forced to mock the transport to be able to receive outgoing messages and check them in tests.

Well, we could use the transport, but we're forced to do something other than read the outgoing messages directly from the gossip module.